IceCubesApp/App/Tabs/Timeline/TimelineTab.swift

```diff
@@ -19,6 +19,8 @@ struct TimelineTab: View {
   @State private var didAppear: Bool = false
   @State private var timeline: TimelineFilter
   @State private var scrollToTopSignal: Int = 0
+  
+  @AppStorage("timelineFilter") public var timelineFilter: String = TimelineFilter.home.title
```

Comment:
should be named something like `last_timeline_filter`, same for the variable name, `lastTimelineFilter` right now it could be a bit confusing with the actual filter enum.